fix :  Inserted images/videos aren't considred as a content in news - EXO-65311

sofyenne opened the pull request:
Before this change, when a user inserted an image or a video into the body of the news during the creation or modification process, the post button would be disabled. After this change, the post button is enabled when the  news body isn't empty and the update button is enabled when a new content is inserted.

aycherif committed:
TASK-62301: Create FB maintenance and update projects versions/dependencies

sofyenne committed:
fix : enable to display published news images selected from existing upload for non members - EXO-65111 (#874)

This change is going to display the published news images selected from existing upload for non space members.

sofyenne committed:
fix : enable to display published news images selected from existing upload for non members - EXO-65111 (#875)

sofyenne committed:
fix :  Inserted images/videos aren't considred as a content in news - EXO-65311

ahamdi commented on webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:
```diff
@@ -395,23 +395,29 @@ export default {
     },
+    hasMediaContent() {
+      const parsedDocument = new DOMParser().parseFromString(this.news.body, 'text/html');
+      const mediaElements = parsedDocument.querySelectorAll('img, video, iframe');
```
Unfortunately this is still insufficient as it is limited to those tags and does not cover all possible HTML tags that could be added (svg, canva, audio, etc ...) even if it not the case now, but everytime a new tag is allowed, we will have to update the fix.
I prefer doing comparison of raw HTML content

akhanfir committed:
fix:[news] search - paste filtered url in a new tab glitches the display - EXO-65200 (#906)

Prior to this change, when create news in three different space, go to news app, open hamburger drawer besides posted articles
unselect all spaces and select only one space , copy the UI url and paste it in a new taw on notice the filter glitches, all news are displayed. After this change, the filter is already mentioned in the url.

sofyenne committed:
updating fix to be able to detect any change

exo-swf commented:
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

sofyenne committed:
Update the fix by removing the usage of the jQuery framework.

Jihed525 committed:
Merge branch 'feature/maintenance' into Task/65311
